backend/tests/openapi.rs

```diff
@@ -0,0 +1,10 @@
+use hack4krak_backend::setup_actix_app;
+use std::fs::read_to_string;
+
+#[test]
```

Comment:
According to rust standards, things in `tests/` are integration tests for your code. I really don't like that here we should check if file in other place is updated. I think better thing would be to add new binary to Cargo.toml:
```toml
[[bin]]
name = "openapi_check"
path = "src/bin/openapi_check.rs"
```
Or at least mark it with full test suite